packages/core-bridge/ai-rules.md

```diff
@@ -0,0 +1,418 @@
+# Bridge Layer Coding Norms, Best Practices and Pertinent Knowledge
```

Comment:
WIP - I'm leaving this doc here even though it is incomplete (I'll complete at a later time, post-merge). I think it still adds some value in its present form. Feel free to read and comment if curious, but don't feel obliged.